app/livechat/server/api/lib/departments.js

```diff
@@ -55,10 +55,8 @@ export async function findDepartmentById({ userId, departmentId, includeAgents =
 
 	const result = {
 		department: await LivechatDepartment.findOne(query),
+		agents: includeAgents && canViewLivechatDepartments ? await LivechatDepartmentAgents.find({ departmentId }).toArray() : undefined,
```

Comment:
```suggestion
		...includeAgents && canViewLivechatDepartments && {
			agents: await LivechatDepartmentAgents.find({ departmentId }).toArray(),
		},
```

Suggesting something even fancier than ternary 😛